src/phoenix/server/api/input_types/CreateTraceAnnotationsInput.py

```diff
@@ -0,0 +1,16 @@
+from typing import Optional
+
+import strawberry
+from strawberry.relay import GlobalID
+from strawberry.scalars import JSON
+
+
+@strawberry.input
+class CreateTraceAnnotationsInput:
```

Comment:
Nit: This name sounds like it should be the top-level input type to the resolver rather than a list of these types.